pyproject.toml

```diff
@@ -56,6 +73,33 @@ plugins = ["pydantic.mypy"]
 [tool.ruff]
 line-length = 120
 
+[tool.ruff.lint]
```

Comment:
Added a ruff config that I used for checking my modified files. I can remove it if required

Reply:
Check with fern the implication of this. Are you ruff checking all files in the SDK including the autogenerated ones?

Reply:
No, I've only applied the linter on my modifications

Reply:
Update: I am removing the section from .toml since Fern has indicated us that they auto-generate the pyproject.toml and we shouldn't manually edit it